src/main/java/com/example/myKitchenManager/db/DBConnectionFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.example.myKitchenManager.db;
+
+import com.example.myKitchenManager.mysqldb.MySQLConnection;
+
+public class DBConnectionFactory {
```

Comment:
We can leverage existing library that already build the DB operation. These libraries are more powerful and fully tested. Check "Java Hibernate" for example.